sdn_tests/testplans/Stress_Latency_Test_Plan.md

```diff
@@ -0,0 +1,385 @@
+
+The robustness is validated through the following methods
+
+-   Monitoring the CPU load for the gNMI server
+-   Validation for the any unwanted process termination during the test
+-   Monitoring the memory usage of the system during the test
+-   Validate that the responses are received in the client from the gNMI server.
+
+## SET UPDATE operation
+
+### Expectation
+
+The expectation is for the gNMI server to process the valid update request and populate the leaf nodes with the values specified in the request payload and return the valid response to the client. If the leaf nodes are not present in the payload then the server should not populate them with the default values and leave those leaf values as-is(not modify them). The gNMI specification description can be found [here](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#344-modes-of-update-replace-versus-update).
+
+## SET REPLACE operation
+
+### Expectation
+
+The expectation is for the gNMI server to process the valid replace request and populate the leaf nodes with the values specified in the request payload and return the valid response to the client. If the leaf nodes are not present in the payload then the server should populate them with the default values. The gNMI specification description can be found [here](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#344-modes-of-update-replace-versus-update).
+
+## SET DELETE operation
+
+### Expectation
+
+The expectation is for the gNMI server to process the valid delete request and remove those nodes from the system. The server should send a valid response to the client once the request has been processed successfully.
+
+## GET operation
+
+### Expectation
+
+The GET response should contain the value of the request nodes. The expectation is for the gNMI server to process the valid get request and should send a valid response containing the values of the request nodes. Only the PROTO and JSON encodings are currently supported in the GPINS.
+
+## Subscribe operation
+
+### Expectation
+
+-   The system should be able to handle concurrent subscribe requests with different modes (ONCE/POLL/STREAM).
+-   The system should be able to reject invalid path( unsupported path )subscriptions. When the gNMI server implements the feature to silently accept the subscription for the unsupported path, the expectation for the unsupported path will change. This feature might be implemented in Q4 21.
```

Comment:
Everything on this line after the first sentence can be removed. Starting with "When the gNMI server...".